net/port.go

```diff
@@ -46,6 +46,25 @@ func GetFreePort() (Port, error) {
 	return Port(l.Addr().(*net.TCPAddr).Port), nil
 }
 
+// GetNextPort returns the immediate next port if it's available.
+func GetNextPort(port string) (Port, error) {
```

Comment:
```suggestion
func NextFreePort(port string) (Port, error) {
```

usage wise its correct 

```
xnet.NextFreePort()
```

Reply:
We have another function name called `GetFreePort()`. To follow the same can I named as `GetNextFreePort()`

Reply:
yeah